README.md

```diff
@@ -40,12 +40,11 @@ Use conda to install Fiona and GDAL:
 
 #### Mac
 
-- `conda create -n [env_name] python=3.8`
-- `conda install -c conda-forge proj=7.0.0`
-- `conda install -c conda-forge pyproj=2.6.0`
+- `conda create -n [env_name] python=3.9`
```

Comment:
Updating the Python version may have implications elsewhere in the code, but as long as it works it works. Was the demo notebook .ipynb tested after this installation?

Reply:
Chris tested this on his Mac, but not using the notebook just by using the package.